src/styles/box.scss

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
please isolate those styles by using SCSS modules, so add `modules` to the filename